### PR TITLE
remove art and timezone (java-only repo)

### PR DESCRIPTION
**Pull request by Vince1171**
Change-Id: I478153bb83a1df985bca0f4a076f90a5a51a975f

### CODE REVIEW

**Comment by Vince1171** [comment]
Btw, we should now delete [the art repo](https://github.com/Halium/android_art) and the [timezone repo](https://github.com/Halium/android_system_timezone)

File: default.xml [hunk 1]
```diff
@@ -725,7 +727,9 @@
 -->
   <project path="system/security" name="LineageOS/android_system_security" groups="pdk" />
   <project path="system/sepolicy" name="android_system_sepolicy" groups="pdk" remote="halium" />
+<!-- java-only repo
```
Comment (Herrie82) [inline default.xml:730]: tzdatacheck is actually being used in init (it wasn't disabled in 7.1 so I kept it in my 8.1 work as well: https://github.com/Halium/android_system_core/blob/halium-8.1/rootdir/init.rc#L524). See my suggestions for this in https://github.com/Herrie82/android_system_timezone/commit/cb536d1f58a659d84db75ee4d6471477e6d64df6 where I removed the Java bits and kept the non-Java for tzdatacheck

File: default.xml [hunk 2]
```diff
@@ -63,7 +63,9 @@
     <linkfile src="root.bp" dest="Android.bp" />
     <linkfile src="bootstrap.bash" dest="bootstrap.bash" />
   </project>
+<!-- removed because it's java stuff
   <project path="art" name="android_art" groups="pdk" remote="halium" />
```
Comment (Herrie82) [inline default.xml:67]: Removing art completely also means removing libsigchain which is being used by (at least) libsurfaceflinger it seems. Seems that building of libsurfaceflinger was disabled in 7.1 as well, so I guess we're good to disable it in 8.1 as well. I'll patch that in the android_frameworks_native similar to what was done for 7.1: https://github.com/Halium/android_frameworks_native/commit/9a547f8e95da7c866b479d9e776d2bae3d2cf78c#diff-d753eba7f87fba97856e0ca871ed5000

**Comment by Herrie82** [comment]
@Vince1171 I removed art and pointed to the new android_system_timezone repo in default.xml. Therefore closing this one now, since it's included in the build. https://github.com/Halium/android/commit/4cb7cdbb516edf836f10f45a4ad9705809be731d

**Pull request closed without merging**